chore(*): docker read-only mode docs (#5273)
Kong/docs.konghq.com: app/_src/gateway/install/docker/index.md

docker network rm kong-net
```

## Running Kong in read-only mode

> To run Kong in read-only mode, mount a Docker volume to the locations where Kong needs to write data. The default configuration requires write access to `/tmp` and to the prefix path, as provided by the following example:

Reviewer: This needs a mention of "starting in 3.2.0". We offer no guarantees for previous versions.

Author: done. @Kong/docs-maintainers should this be handled with some special version tagging?

> Starting with version 3.2.0 of {{site.base_gateway}} it is possible to run the container in read-only mode. To do so, mount a Docker volume to the locations where Kong needs to write data. The default configuration requires write access to `/tmp` and to the prefix path, as provided by the following example:

Reviewer 2: ```suggestion
Starting with {{site.base_gateway}} 3.2.0, you can run the container in read-only mode. To do so, mount a Docker volume to the locations where Kong needs to write data. The default configuration requires write access to `/tmp` and to the prefix path:
```
